FSharp/lib/editor.py

```diff
@@ -56,6 +79,9 @@ def refresh(self, fs_file):
         assert isinstance(fs_file, FSharpFile), 'wrong argument: %s' % fs_file
         # todo: run in alternate thread
 
+        if fs_file.is_script_file:
+            return
+
```

Comment:
Could this method (`refresh`) be better named to indicate that it sets active project? And in the case of a script file or the case that there is no project file for a .fs file, shouldn't we 'unset' the project in fsac so we don't get wrong results?